modules/monitoring-supported-remote-write-authentication-settings.adoc

```diff
@@ -30,285 +28,3 @@ You cannot use this method simultaneously with authorization, AWS Signature Vers
-data:
-  config.yaml: |
-    prometheusK8s:
-      remoteWrite:
-      - url: "https://test.example.com/api/write"
-        oauth2:
-          clientId:
-            secret:
-              name: oauth2-credentials <1>
-              key: id <2>
-          clientSecret:
-            name: oauth2-credentials <1>
-            key: secret <2>
-          tokenUrl: https://example.com/oauth2/token <3>
-          scopes: <4>
-          - <scope_1>
-          - <scope_2>
-          endpointParams: <5>
-            param1: <parameter_1>
-            param2: <parameter_2>
-----
-<1> The name of the corresponding `Secret` object. Note that `ClientId` can alternatively refer to a `ConfigMap` object, although `clientSecret` must refer to a `Secret` object.
-<2> The key that contains the OAuth 2.0 credentials in the specified `Secret` object.
-<3> The URL used to fetch a token with the specified `clientId` and `clientSecret`.
-<4> The OAuth 2.0 scopes for the authorization request. These scopes limit what data the tokens can access.
-<5> The OAuth 2.0 authorization request parameters required for the authorization server.
-
-.Sample YAML for TLS client authentication
-
-The following shows sample TLS client settings for a `tls` `Secret` object named `mtls-bundle` in the `openshift-monitoring` namespace.
-
-[source,yaml]
-----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: mtls-bundle
-  namespace: openshift-monitoring
-data:
-  ca.crt: <ca_cert> <1>
-  client.crt: <client_cert> <2>
-  client.key: <client_key> <3>
-type: tls
-----
-<1> The CA certificate in the Prometheus container with which to validate the server certificate.
-<2> The client certificate for authentication with the server.
-<3> The client key.
-
-The following sample shows a `tlsConfig` remote write authentication configuration that uses a TLS `Secret` object named `mtls-bundle`.
-
-[source,yaml]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cluster-monitoring-config
-  namespace: openshift-monitoring
-data:
-  config.yaml: |
-    prometheusK8s:
-      remoteWrite:
-      - url: "https://remote-write-endpoint.example.com"
-        tlsConfig:
-          ca:
-            secret:
-              name: mtls-bundle <1>
-              key: ca.crt <2>
-          cert:
-            secret:
-              name: mtls-bundle <1>
-              key: client.crt <3>
-          keySecret:
-            name: mtls-bundle <1>
-            key: client.key <4>
-----
-<1> The name of the corresponding `Secret` object that contains the TLS authentication credentials. Note that `ca` and `cert` can alternatively refer to a `ConfigMap` object, though `keySecret` must refer to a `Secret` object.
-<2> The key in the specified `Secret` object that contains the CA certificate for the endpoint.
-<3> The key in the specified `Secret` object that contains the client certificate for the endpoint.
-<4> The key in the specified `Secret` object that contains the client key secret.
```

Comment:
This content is still accurate, so it should still be there. The reason it's appearing as deleted is because I split this module into two: monitoring-supported-remote-write-authentication-settings and monitoring-example-remote-write-authentication-settings. So the "deleted" content was just moved to the new monitoring-example-remote-write-authentication-settings module.